1-Initial-Setup/1.1-Filesystem-Configuration.bats

```diff
@@ -0,0 +1,203 @@
+    run bash -c "findmnt -n /tmp | grep -v noexec"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.6 Ensure /dev/shm is configured (Automated)" {
+    run bash -c "findmnt -n /dev/shm"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "/dev/shm "* ]]
+}
+
+@test "1.1.7 Ensure nodev option set on /dev/shm partition (Automated)" {
+    run bash -c "findmnt -n /dev/shm | grep -v nodev"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.8 Ensure nosuid option set on /dev/shm partition (Automated)" {
+    run bash -c "findmnt -n /dev/shm | grep -v nosuid"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.9 Ensure noexec option set on /dev/shm partition (Automated)" {
+    run bash -c "findmnt -n /dev/shm | grep -v noexec"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.10 Ensure separate partition exists for /var (Automated)" {
+    run bash -c "findmnt /var"
+    [ "$status" -eq 0 ]
+    [[ "$output" = " /var "* ]]
+}
+
+@test "1.1.11 Ensure separate partition exists for /var/tmp (Automated)" {
+    run bash -c "findmnt /var/tmp"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "/var/tmp "* ]]
+}
+
+@test "1.1.12 Ensure /var/tmp partition includes the nodev option (Automated)" {
+    run bash -c "findmnt -n /var/tmp | grep -v nodev"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.13 Ensure /var/tmp partition includes the nosuid option (Automated)" {
+    run bash -c "findmnt -n /var/tmp | grep -v nosuid"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.14 Ensure /var/tmp partition includes the noexec option (Automated)" {
+    run bash -c "findmnt -n /var/tmp | grep -v noexec"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.15 Ensure separate partition exists for /var/log (Automated)" {
+    run bash -c "findmnt /var/log"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "/var/log "* ]]
+}
+
+@test "1.1.16 Ensure separate partition exists for /var/log/audit (Automated)" {
+    run bash -c "findmnt /var/log/audit"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "/var/log/audit "* ]]
+}
+
+@test "1.1.17 Ensure separate partition exists for /home (Automated)" {
+    run bash -c "findmnt /home"
+    [ "$status" -eq 0 ]
+    [[ "$output" = "/home "* ]]
+}
+
+@test "1.1.18 Ensure /home partition includes the nodev option (Automated)" {
+    run bash -c "findmnt -n /home | grep -v nodev"
+    [ "$status" -eq 1 ]
+    [ "$output" = "" ]
+}
+
+@test "1.1.19 Ensure nodev option set on removable media partitions (Manual)" {
+    skip "This audit has to be done manually"
+}
+
+@test "1.1.20 Ensure nosuid option set on removable media partitions (Manual)" {
+    skip "This audit has to be done manually"
+}
+
+@test "1.1.21 Ensure noexec option set on removable media partitions (Manual)" {
+    skip "This audit has to be done manually"
+}
+
+@test "1.1.22 Ensure sticky bit is set on all world-writable directories (Automated)" {
```

Comment:
For me, this test does not return anything (as desired), but the RC is 123. If necessary, the test must be executed with additional permissions.
`sudo bash -c 'df --local -P | awk '\''{if (NR!=1) print $6}'\'' | xargs -I '\''{}'\'' find '\''{}'\'' -xdev -type d \( -perm -0002 -a ! -perm -1000 \) 2>/dev/null'` worked for me.

Reply:
I think we can assume that all tests are run with root permissions, especially those who check the whole system.